e2e_test/batch/basic/table_with_default_columns.slt.part

```diff
@@ -43,8 +43,17 @@ select * from t2;
 1 2
 2 2
 
+statement ok
+create table tt (v1 int, v2 timestamptz default now());
+
+statement ok
+insert into tt values (1);
```

Comment:
We can further check that the value of `v2` is greater than 2021.

https://github.com/risingwavelabs/risingwave/blob/49a5a9f4d21d48bf50345ada94acafb6eb35997c/e2e_test/ddl/table/generated_columns.slt.part#L81-L109